1.5/Documentation/Advanced-Concepts/Activation-Garbage-Collection.html

```diff
@@ -0,0 +1,231 @@
+﻿<!DOCTYPE html>
+<!--[if IE]><![endif]-->
```

Comment:
still html file here

Reply:
or is this supposed to be html files since it is 1.5 docs? If yes, suggest separate 1.5 docs and 2.0 docs into 2 pr to help review. This PR is too big I even have trouble loading.